components/picker/stories/template.js

```diff
@@ -76,7 +72,22 @@ export const Picker = ({
 					customClasses: ["spectrum-Picker-icon"],
 				}, context))
 			}
-			<span class="${rootClass}-label is-placeholder">${placeholder}</span>
+			${when(showWorkflowIcon, () =>
+				Icon({
+					size,
+					setName: "workflow",
+					iconName: "Image",
```

Comment:
Is there any concern over this being hardcoded to `Image`? I can make a ticket to refactor this further and create a leading icon control or something.

Reply:
I don't think so but we can always revisit if we need to later!